dts/bindings/sensor/maxim,max30101.yaml

```diff
@@ -1,8 +1,250 @@
+        then the flag is set when the 30th word is written to the FIFO).
+        Range: 0-15, Default: 0, same as after Power Reset
+    fifo-rollover-en:
+      type: boolean
+      required: false
+      description: |
+        Controls the behavior of the FIFO when the FIFO becomes completely
+        filled with data. If set, the FIFO address rolls over to zero and the
+        FIFO continues to fill with new data. If not set, then the FIFO is
+        not updated until FIFO_DATA is read or the WRITE/READ pointer
+        positions are changed.
+    led-mode:
+      type: int
+      required: false
+      default: 2
+      enum:
+       - 2
+       - 3
+       - 7
+      description: |
+        Set the operation mode of the MAX30101.
+        2 = heart rate mode, only red led
+        3 = spo2 mode, red and ir led
+        7 = multi led mode, leds according configuration of slots
+        Default: 2, sufficient for heart rate recognition
+    led1-pa:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Set the pulse amplitude to control the LED1 (red) current. The actual
+        measured LED current for each part can vary significantly due to the
+        trimming methodology.
+        0x00 = 0.0 mA
+        0x01 = 0.2 mA
+        0x02 = 0.4 mA
+        0x0f = 3.1 mA
+        0xff = 50.0 mA
+        Default: 0xff, full brightness for data quailty
+    led2-pa:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Set the pulse amplitude to control the LED2 (IR) current. The actual
+        measured LED current for each part can vary significantly due to the
+        trimming methodology.
+        0x00 = 0.0 mA
+        0x01 = 0.2 mA
+        0x02 = 0.4 mA
+        0x0f = 3.1 mA
+        0xff = 50.0 mA
+        Default: 0xff, full brightness for data quailty
+    led3-pa:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Set the pulse amplitude to control the LED3 (green) current. The actual
+        measured LED current for each part can vary significantly due to the
+        trimming methodology.
+        0x00 = 0.0 mA
+        0x01 = 0.2 mA
+        0x02 = 0.4 mA
+        0x0f = 3.1 mA
+        0xff = 50.0 mA
+        Default: 0xff, full brightness for data quailty
+    adc-rge:
+      type: int
+      required: false
+      default: 2
+      enum:
+       - 0
+       - 1
+       - 2
+       - 3
+       - 4
+       - 5
+       - 6
+       - 7
+      description: |
+        Set the ADC's full-scale range.
+        0 = 7.81 pA/LSB
+        1 = 15.63 pA/LSB
+        2 = 31.25 pA/LSB
+        3 = 62.5 pA/LSB
+        Default: 2, compromise between saturation and resolution
+    smp-rate-ctrl:
+      type: int
+      required: false
+      default: 0
+      enum:
+       - 0
+       - 1
+       - 2
+       - 3
+       - 4
+       - 5
+       - 6
+       - 7
```

Comment:
ditto. 
I think is more clear to add the enum values description in a comment for each line above.
Please do it for all.